lib/rspec/core/configuration.rb

```diff
@@ -1412,13 +1415,32 @@ def self.delegate_to_ordering_manager(*methods)
       #     end
       #   end
       #
-      #   describe MyClass, :order => :reverse do
+      #   RSpec.describe 'MyClass', :order => :reverse do
       #     # ...
       #   end
       #
       # @note Pass the symbol `:global` to set the ordering strategy that
       #   will be used to order the top-level example groups and any example
       #   groups that do not have declared `:order` metadata.
+      #
+      # @example
+      #   RSpec.configure do |rspec|
+      #     rspec.register_ordering :global do |examples|
+      #       acceptance, other = examples.partition do |example|
+      #         example.metadata[:type] == :acceptance
```

Comment:
I'm more interested in getting a good API here so there's nothing to blog about.

If the API is to stay this "complex", I'd rather create a PR where the strategy can be set from the command-line (like a custom formatter). Otherwise people are going to need unit tests for their RSpec config...

Just saying that to me this is code smell of some sort.